src/axom/quest/SamplingShaper.hpp

```diff
@@ -432,16 +232,45 @@ class SamplingShaper : public Shaper
     }
 
     SLIC_INFO(
-      axom::fmt::format("{:-^80}",
-                        axom::fmt::format(" Querying the octree for shape '{}'", shape.getName())));
+      axom::fmt::format("{:-^80}", axom::fmt::format(" Querying for shape '{}'", shape.getName())));
 
     switch(getShapeDimension())
     {
     case klee::Dimensions::Two:
-      runShapeQueryImpl(m_inoutSampler2D);
+      runShapeQueryImpl(m_inoutSampler2D.get());
       break;
     case klee::Dimensions::Three:
-      runShapeQueryImpl(m_inoutSampler3D);
+      if(this->shapeFormat(shape) == "stl")
```

Comment:
Are "stl" and "proe" checks done to determine whether there are triangles vs tets? If so, can this be queried from the shape so file format checks do not proliferate?

Reply:
Since the templates are baked into the sampler type, 
I think this whole block will look like the following w/ `std::variant`:
```cpp
    std::visit([](auto& inst){ runShapeQueryImpl(inst.get(); }, m_sampler);
```